lib/fluttertoast_web.dart

```diff
@@ -69,27 +69,29 @@ class FluttertoastWebPlugin {
   /// [injectCssAndJSLibraries] which add the JS and CSS files into DOM
   Future<void> injectCssAndJSLibraries() async {
     final List<Future<void>> loading = <Future<void>>[];
-    final List<html.HtmlElement> tags = <html.HtmlElement>[];
+    final List<web.HTMLElement> tags = <web.HTMLElement>[];
 
     final cssUrl = ui.assetManager.getAssetUrl(
       'packages/fluttertoast/assets/toastify.css',
     );
-    final html.LinkElement css = html.LinkElement()
+    final web.HTMLLinkElement css = web.HTMLLinkElement()
       ..id = 'toast-css'
-      ..attributes = {"rel": "stylesheet"}
+      ..setAttribute("rel", "stylesheet")
       ..href = cssUrl;
     tags.add(css);
-    
+
     final jsUrl = ui.assetManager.getAssetUrl(
       'packages/fluttertoast/assets/toastify.js',
     );
-    final html.ScriptElement script = html.ScriptElement()
+    final web.HTMLScriptElement script = web.HTMLScriptElement()
       ..async = true
       // ..defer = true
       ..src = jsUrl;
     loading.add(script.onLoad.first);
     tags.add(script);
-    html.querySelector('head')!.children.addAll(tags);
+    for (final web.HTMLElement tag in tags) {
+      web.document.querySelector('head')!.append(tag);
+    }
```

Comment:
querySelector has children, why for loop?

Reply:
Can you elaborate on how querySelector having children can help us get rid of the for loop? Maybe a code snippet?

Reply:
nevermind, you are right